# Command to send to the robot
std_msgs/Header header

# Velocity commands
geometry_msgs/Twist velocity_command

# Joint position commands
float32[] joint_positions

# Control mode
string control_mode